docs/developer-manual/consume.md

```diff
@@ -4,31 +4,228 @@ sidebar_position: 1
 slug: /consume
 ---
 
-# Consume fact statements
+# Consuming Orcfax Statements
 
-## Important Notice
+This document aims to explain how dapp developers can integrate Orcfax feeds into their dapp
+with all the gory (technical) details included.
 
-This page is being updated to reflect the upcoming changes in the Orcfax v1
-solution. Once this upgrade has been released, instructions on how to consume
-the enhanced datum will be made available here.
+## Overview
 
-Historical data concerning the v0 Orcfax solution can be found in
-[Archived resources](archived-resources) and information pertaining to policy
-ID's in [Policies](policies).
+### Orcfax
 
-## Fact statements
+Orcfax is a Cardano native oracle platform.
 
-A fact statement is a datum that the Orcfax network publishes to a Cardano
-blockchain transaction. This datum can then be used as an input to Cardano smart
-contracts and dApps.
+Orcfax is organized into feeds.
+Generally a feed concerns a time series data, such as the exchange rate from ada to usd.
+A feed is a data pipeline that specifies how data is sourced, processed, and the format of its outputs.
+FIXME : See the orcfax explorer for more details on the feeds available and their definitions.
 
-## UTxO model
+One part of the overall protocol is getting the data on-chain to be consumable by dapps and their end users.
+The part of a feed's output that ends up on-chain is called a **statement**.
+A statement is included in the datum of an input, and thus available to dapps via the reference input field.
```

Comment:
```suggestion
A statement is included in the datum of a transaction input, and thus available to dapps via reference input.
```

Also, maybe reference to CIP-0031?

Reply:
I think its so old and established, it can be taken as known. 
(As in it would be better to just point at Plutus V2 spec. I've added in `V2` to make it clear we are V2)